lightning/Cargo.toml

```diff
@@ -56,7 +56,7 @@ regex = "1.5.6"
 [dev-dependencies.bitcoin]
 version = "0.29.0"
 default-features = false
-features = ["bitcoinconsensus", "secp-recovery"]
```

Comment:
I believe `bitcoinconsensus` is still required here, maybe you didn't intend to remove it?

Reply:
it wouldn't compile in wasm for me with this, added back to fix CI. probably something local for me